indico/modules/events/api.py

```diff
@@ -538,6 +512,33 @@ def _update_query(self, query):
     def serialize_events(self, events):
         return map(self._build_event_api_data, events)
 
+    def _serialize_category_path(self, category):
+        visibility = {'id': '', 'name': ''}
+        path = [self._serialize_path_entry(category_data) for category_data in category.chain]
+        if category.visibility is None:
+            visibility['name'] = 'Everywhere'
```

Comment:
Very good point!
However, I cannot just do `visibility = path[...]` cause the path element has much more info. I should do this instead:
```python
try:
    visibility['id'] = path[-category.visibility]['id']
    visibility['name'] = path[-category.visibility]['name']
```